org/urls.py

```diff
@@ -4,15 +4,13 @@
 
 urlpatterns = [
     path('', OrgView.as_view()),
-    path('<int:org_id>/volunteer/',AddVolunteer,name='add_volunteer'),
-    path('<int:org_id>/group/', GroupView.as_view()),
-    path('<int:org_id>/', EditOrg, name='edit_org'),
+    path('<int:org_id>/', EditOrgView.as_view(), name='edit_org'),
```

Comment:
```suggestion
    path('<int:org_id>/', OrgView.as_view(), name='edit_org'),
```
Please rename the class